module/solutions/helper.js

```diff
@@ -84,4 +91,362 @@ module.exports = class SolutionsHelper {
     });
   }
 
+     /**
+   * Create solution.
+   * @method 
+   * @name createSolution
+   * @param {Object} data - solution creation data.
```

Comment:
@aman-tunerlabs - rename it to solution data, mention all the keys in comments here. Do likewise in all places as required.